src/test/Components/CreateImageWizard/CreateImageWizard.compliance.test.js

```diff
@@ -5,7 +5,7 @@ import '@testing-library/jest-dom';
 import { screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
-import CreateImageWizard from '../../../Components/CreateImageWizard/CreateImageWizard';
+import CreateImageWizard from '../../../Components/CreateImageWizardV2/CreateImageWizard';
```

Comment:
These changes should probably be in the `src/test/Components/CreateImageWizardV2/CreateImageWizard.compliance.test.tsx` file.

Reply:
I think I chagned it :)

Reply:
Oh I mean that you made the changes in the CreateImageWizard version of the test

![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/27567f73-23f0-4547-bcf6-9ea8d08cae7b)

Reply:
ye, sure you correct